app/aggregates/reviewing/command.rb

```diff
@@ -19,5 +19,11 @@ def repository
     def stream_name
       "Reviewing$#{application_id}"
     end
+
+    class << self
```

Comment:
Just thinking we could have a "method not implemented" stubbed call method in the base class perhaps it would signpost that commands are abstract

```
def call
  raise MethodNotImplemented
end
```

Small thing but it might be easier to undersand that the call is not defined here but in the children.